aries-site/src/components/cards/PreviewCard.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box } from 'grommet';
+
+export const PreviewImageCard = ({ children, ...rest }) => {
+  return (
+    <Box
+      height="small"
+      round="xsmall"
+      overflow="hidden"
+      style={{ position: 'relative' }}
+      {...rest}
+    >
+      {children}
+    </Box>
+  );
+};
+
+PreviewImageCard.defaultProps = {
+  background: 'background-back',
+};
+
+PreviewImageCard.propTypes = {
+  children: PropTypes.node,
+  background: PropTypes.string,
```

Comment:
Do we want to accept { dark: 'valeu', light: 'value' } as a type for background?  If so,
```suggestion
  background: PropTypes.oneOfType([
      PropTypes.string,
      PropTypes.shape({
          dark: PropTypes.string,
          light: PropTypes.string,
      }),
  ]),
```

Reply:
I think since for background we are using either `background-front` or `background-back` which already have their own light and dark in the theme it is fine. to just leave it as. one string being passed through. That was my though process.

Reply:
I'm okay with this for now. We can modify in the future if it is too restrictive.